opm/parser/eclipse/EclipseState/Schedule/Network/ExtNetwork.hpp

```diff
@@ -38,14 +38,19 @@ class ExtNetwork {
     ExtNetwork() = default;
     bool active() const;
     bool has_node(const std::string& name) const;
+    bool has_indexed_node_name(std::string name) const;
```

Comment:
The functions `has_indexed_node_name()` and `add_indexed_node_name()` should be private, and they should be called internally from the `add_branch()` method.

Reply:
OK - fine!